cmd/apply.go

```diff
@@ -29,15 +29,27 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type ApplyOptions struct {
+	Options
+}
+
+var ao = &ApplyOptions{}
+
 // applyCmd represents the apply command
 var applyCmd = &cobra.Command{
-	Use:   "apply [-n|--name NAME]",
+	Use:   "apply <NAME>",
 	Short: "Apply a cluster resource to a cloud",
 	Long: `Use this command to apply an API model in a cloud.
 
 This command will attempt to find an API model in a defined state store, and then apply any changes needed directly to a cloud.
 The apply will run once, and ultimately time out if something goes wrong.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			ao.Name = strEnvDef("KUBICORN_NAME", "")
+		} else {
```

Comment:
Can we check for `> 1` as well and throw an error that too many arguments were passed?